src/Test.UnitTests.Sarif/TestData/RoundTripping/RoundTripping.sarif

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Bug1673Tester"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "TST0001",
+          "message": {
+            "text": "Hello."
+          },
+          "properties": {
+            "date": "2019-09-25T15:07Z",
+            "string": "Hello, string!",
+            "integer": 42,
+            "bigInteger": 5000000000,
+            "true": true,
+            "false": false,
+            "dateArray": [
+  "2019-09-26T15:52Z",
+  "2019-09-26T15:54Z"
+],
```

Comment:
This is how `Formatting.Indented` renders the arrays and sub-objects within `properties`. I don't know why. I wonder if it's related to the actual bug we're trying to ferret out (failure to round-trip certain property bag properties).

Reply:
Oh, it's because the `PropertyBagConverter` has taken over the serialization at this point and it doesn't retain the indentation.

---
In reply to: [328869713](https://github.com/microsoft/sarif-sdk/pull/1688#discussion_r328869713) [](ancestors = 328869713)